server/service/individualServices/ReadInventoryData.js

```diff
@@ -265,18 +310,504 @@ async function formulateEquipmentInfo(equipmentInfoList) {
+          supportedSfpPmd.currentlyOperatedPmd = operatedPmdResponse.currentlyOperatedPmd;
+        }
+      }
+      traceIndicatorIncrementer = operatedPmdResponse.traceIndicatorIncrementer;
+      pluggedSfpPmdList.push(supportedSfpPmd);
+
+    }
+    pluggedSfpPmdListResponse = {
+      pluggedSfpPmdList: pluggedSfpPmdList,
+      traceIndicatorIncrementer: traceIndicatorIncrementer
+    }
+    return pluggedSfpPmdListResponse;
+
+  } catch (error) {
+    console.log(error);
+    return new createHttpError.InternalServerError();
+  }
+}
+
+/**
+ * Filters wire-interface-ltp for EQUIPMENT_CATEGORY = EQUIPMENT_CATEGORY_SMALL_FORMFACTOR_PLUGGABLE
+ * @param {String} mountName Identifier of the device at the Controller
+ * @param {Object} ltpStructure ControlConstruct provided from cache
+ * @param {Object} requestHeaders Holds information of the requestHeaders like Xcorrelator , CustomerJourney,User etc.
+ * @param {Integer} traceIndicatorIncrementer traceIndicatorIncrementer to increment the trace indicator
+ * @returns {Object} returns original-ltp-name(interface-name) value and traceIndicatorIncrementer
+ */
+async function getListOfPluggableSfpLtp(mountName, ltpStructure, requestHeaders, traceIndicatorIncrementer) {
+  let pluggableSfpListResponse = {};
+  let pluggableSfpList = [];
+  const equipmentUuidCallback = "RequestForProvidingAcceptanceDataCausesRetrievingSfpInformation.EquipmentUuid";
+  const equipmentCategoryCallback = "RequestForProvidingAcceptanceDataCausesRetrievingSfpInformation.EquipmentCategory";
+  try {
+    let clientAndFieldParamsForEquipmentUuid = await IndividualServiceUtility.getConsequentOperationClientAndFieldParams(equipmentUuidCallback);
+    let clientAndFieldParamsForEquipmentCategory = await IndividualServiceUtility.getConsequentOperationClientAndFieldParams(equipmentCategoryCallback);
+    let wireInterfaceLtpList = await LtpStructureUtility.getLtpsOfLayerProtocolNameFromLtpStructure(WIRE_INTERFACE.MODULE + WIRE_INTERFACE.LAYER_PROTOCOL_NAME, ltpStructure);
+    for (let i = 0; i < wireInterfaceLtpList.length; i++) {
+      let pathParamList = [];
+      let wireInterfaceLtp = wireInterfaceLtpList[i];
+      let wireInterfaceUuid = wireInterfaceLtp[onfAttributes.GLOBAL_CLASS.UUID];
+      pathParamList.push(mountName, wireInterfaceUuid);
+      /****************************************************************************************************
+      * RequestForProvidingAcceptanceDataCausesRetrievingSfpInformation.EquipmentUuid
+      *   MWDI://core-model-1-4:network-control-domain=cache/control-construct={mount-name}
+      *    /logical-termination-point={uuid}/ltp-augment-1-0:ltp-augment-pac?fields=equipment
+      *****************************************************************************************************/
+      let equipmentUuidResponse = await IndividualServiceUtility.forwardRequest(clientAndFieldParamsForEquipmentUuid, pathParamList, requestHeaders, traceIndicatorIncrementer++);
```

Comment:
Since we are not incrementing the traceindicatorIncrementer in line 365, let linkIdResponse = await getAirInterfaceConfigurationLinkId(mountName, servingPhysicLtp, requestHeaders, traceIndicatorIncrementer);

the post incrementer here will increase the traceIndicator for the next execution , but not the current. Please have a check. Thanks.